### PR TITLE
xfce4-taskmanager: Update to v1.5.7

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Changes:
   - Update copyright year
   - build: Fix `-Wcast-align` warning from clang
   - Fix UBSan errors
   - settings-dialog: Fix `XtmRefreshRate` memory leak
   - Dark mode for process-statusbar
   - Dark mode for process-monitor
   - Allow multiple instances via command line option.
   - Fix blurry app icons when UI scale > 1
   - Use symbolic window picker icon in toolbar
   - Fix broken "show-legend" setting sync
   - Translation Updates: Catalan, Croatian, Estonian, Greek, Hebrew, Italian, Occitan (post  1500), Polish, Spanish, Swedish, Turkish
- Add appstream metainfo (Part of getsolus/packages#1389)

**Test Plan**

Install and launch it in the XFCE VM

**Checklist**

- [x] Package was built and tested against unstable
- [ ] This change could gainfully be listed in the weekly sync notes once merged

### CODE REVIEW

File: packages/x/xfce4-taskmanager/files/org.xfce.taskmanager.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.xfce.taskmanager</id>
+
+  <name>Task Manager</name>
+  <summary>Easy to use task manager </summary>
```
Comment (EbonJaeger) [inline packages/x/xfce4-taskmanager/files/org.xfce.taskmanager.metainfo.xml:6]: Trailing space in the `summary`

File: packages/x/xfce4-taskmanager/files/org.xfce.taskmanager.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+
+  <description>
+    <p>
+      The xfce4-taskmanager component is an easy to use graphical task manager.
+    </p>
+    <ul>
+      <li>Support for Linux, OpenBSD, NetBSD, FreeBSD and OpenSolaris. </li>
```
Comment (EbonJaeger) [inline packages/x/xfce4-taskmanager/files/org.xfce.taskmanager.metainfo.xml:16]: Trailing space in this list item

**Review by EbonJaeger — COMMENTED** [review]
I'm not sure how much it matters, but two of the metainfo lines have a space at the end of the sentence.

**Comment by malfisya** [comment]
It doesn't matter much, but it is nice to tidy. :smile: Fixed.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! Feel free to merge it, I won't be able to tonight, and it would be great to have in for sync tomorrow with all the rest of the XFCE stuff.

**Pull request merged**